local/kcp/start.sh

```diff
@@ -32,15 +32,17 @@ precheck() {
 }
 
 KCP_DIR="${KCP_DIR:-}"
-kcp-binaries () {
+kcp-binaries() {
   if [[ -z "${KCP_DIR}" ]]; then
+    CKCP_DIR="${PARENT_PATH}/../../ckcp"
+    kcp_image_tag="$(yq -e '.images[0].newTag' "$CKCP_DIR"/openshift/overlays/dev/kustomization.yaml)" 
```

Comment:
We don't need that here and having in the script a dependency to ckcp and an openshift overlay does not seem right.
I would just let it as it was. At the end of the day "release-0.5" is just a default value for the kcp branch to use and the developer is free to select a different one.

Reply:
Fixed.